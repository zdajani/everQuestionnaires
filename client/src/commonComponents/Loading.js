import React from 'react';
import './styles/Loading.css';

const Loading = () => (
  <div className="loading-screen">
  	<img src={'https://tools.cisco.com/security/center/Resources/images/loader.gif'} alt="loadig" id="loading"/>
  </div>
);

export default Loading;