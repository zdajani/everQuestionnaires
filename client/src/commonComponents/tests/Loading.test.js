import React from 'react';
import { shallow } from 'enzyme';
import Loading from '../Loading';
import toJson from 'enzyme-to-json';

describe('Loading component', () => {
  it('should render loading page', () => {
    const component = shallow(<Loading />);
    expect(toJson(component)).toMatchSnapshot();
  });
});