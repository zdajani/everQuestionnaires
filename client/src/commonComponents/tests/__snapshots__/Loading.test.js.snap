// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should render loading page 1`] = `
<div
  className="loading-screen"
>
  <img
    alt="loadig"
    id="loading"
    src="https://tools.cisco.com/security/center/Resources/images/loader.gif"
  />
</div>
`;
